CHANGELOG.md

```diff
@@ -28,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Built-in function `address()` now handles parse errors correctly: PR [#357](https://github.com/tact-lang/tact/pull/357)
 - All identifiers in error messages are now quoted for consistency: PR [#363](https://github.com/tact-lang/tact/pull/363)
 - The grammar of the unary operators has been fixed, constant and function declarations are prohibited for contracts and at the top level of Tact modules: PR [#365](https://github.com/tact-lang/tact/pull/365)
+- Typos in ABI generation: : PR [#372](https://github.com/tact-lang/tact/pull/372)
```

Comment:
looks like one `:` is extra here

Reply:
Oops! Well-spotted, will fix in a PR that brushes up the changelog before releasing v1.3.1